src/common/socialShare.js

```diff
@@ -0,0 +1,171 @@
+import React from 'react';
+
+import { useTranslation } from 'react-i18next';
+
+import CloseIcon from '@mui/icons-material/Close';
+import EmailIcon from '@mui/icons-material/Email';
+import FacebookIcon from '@mui/icons-material/Facebook';
+import WhatsAppIcon from '@mui/icons-material/WhatsApp';
+import {
+  Box,
+  Button,
+  IconButton,
+  Modal,
+  Stack,
+  TextField,
+  Typography,
+} from '@mui/material';
+import useMediaQuery from '@mui/material/useMediaQuery';
+
+import theme from 'theme';
+
+const SocialShare = ({ item }) => {
+  const { t } = useTranslation();
+  const [open, setOpen] = React.useState(false);
+  const [buttonCopied, setButtonCopied] = React.useState(false);
+  const handleOpen = () => setOpen(true);
+  const handleClose = () => setOpen(false);
+  const isSmall = useMediaQuery(theme.breakpoints.down('sm'));
+
+  const pageUrl = window.location;
+
+  const shareViaEmail = () => {
+    window.open('mailto:?body=' + pageUrl);
+  };
+
+  const shareViaWhatsApp = () => {
+    window.open('https://wa.me/?text=' + pageUrl);
+  };
+
+  const shareViaFacebook = () => {
+    window.open('http://www.facebook.com/share.php?u=' + pageUrl);
+  };
+
+  const copyToClipboard = () => {
+    navigator.clipboard.writeText(pageUrl);
+    setButtonCopied(true);
+  };
+
+  return (
+    <>
+      <Button variant="outlined" onClick={handleOpen}>
+        {t('share.button')}
+      </Button>
+      <Modal
+        open={open}
+        onClose={handleClose}
+        aria-labelledby="modal-modal-title"
+        aria-describedby="modal-modal-description"
+      >
+        <Box sx={theme.modalStyle}>
+          <IconButton
+            onClick={handleClose}
+            sx={{
+              position: 'absolute',
+              top: theme.spacing(1),
+              right: theme.spacing(1),
+            }}
+          >
+            <CloseIcon />
+          </IconButton>
+
+          <Typography
+            variant="h1"
+            component="h2"
+            sx={{ paddingTop: 0, marginBottom: theme.spacing(4) }}
+          >
+            {t(`share.${item}`)}
+          </Typography>
+          <Typography sx={{ marginBottom: theme.spacing(1) }}>
+            {t('share.services')}
+          </Typography>
+          <Stack
+            direction={isSmall ? 'column' : 'row'}
+            justifyContent="space-between"
+          >
+            <Button
+              variant="outlined"
+              startIcon={<EmailIcon sx={{ paddingRight: '5px' }} />}
+              onClick={shareViaEmail}
+            >
+              Email
+            </Button>
+            <Button
```

Comment:
The responsiveness of this component is not working properly:
![image](https://user-images.githubusercontent.com/348237/169904380-8cee7562-03a0-4558-8634-b81ab7e4be1f.png)